avm/ptn/azd/container-app/main.bicep

```
@@ -0,0 +1,235 @@
+metadata name = 'avm/ptn/azd/container-app'
+metadata description = 'Creates a container app in an Azure Container App environment.'
+metadata owner = 'Azure/module-maintainers'
+
+@description('Required. The name of the Container App.')
+param containerAppName string
```

```suggestion
param name string
```

The ptn itself is called 'Container-App'. As such it should ideally not be necessary to also have it mentioned in the name parameter of the module